docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
After updating context to ```../``` I just moved the file to ```docker/docker-compose.yml``` and executed: 
```
cd docker
docker-compose build
docker-compose run --rm --service-ports ms
```

This results in a console prompt in root folder context.

I will accept either way on location, isolation of docker related files is nice, however being able to start the container from the root of the tree is nice too.

Which ever you feel is more simple for users to consume works.